activerecord/CHANGELOG.md

```diff
@@ -1,3 +1,18 @@
+*   Add `set_constraints` and `defer_constraints` helpers for PostgreSQL
+
+    ```ruby
+       Post.create!(user_id: -1) # => ActiveRecord::InvalidForeignKey
+
+       Post.connection.defer_constraints do
+         p = Post.create!(user_id: -1)
+         u = User.create!
+         p.user = u
+         p.save!
+       end
+    ```
```

Comment:
```suggestion
    ```ruby
     Post.create!(user_id: -1) # => ActiveRecord::InvalidForeignKey

     Post.connection.defer_constraints do
       p = Post.create!(user_id: -1)
       u = User.create!
       p.user = u
       p.save!
     end
    ```
```

Reply:
Actually looks like 3 spaces, I guess to match the fenced backticks? You can just indent them inline. :bow: